src/api.cpp

```diff
@@ -3493,6 +3493,55 @@ cl_int CLVK_API_CALL clEnqueueNDRangeKernel(
         }
     }
 
+    // Try to pick a sensible work-group size if the user didn't specify one.
+    if (local_work_size == nullptr) {
+        lws[0] = lws[1] = lws[2] = 1;
```

Comment:
The initialiser above will have done that.

Reply:
Agreed. I've left this in now that I've moved the code into `cvk_device` though, to avoid assuming that the caller has done that initialization.

Reply:
Makes sense.